applications/openshift/api-server/api_server_admission_control_plugin_AlwaysAdmit/rule.yml

```diff
@@ -4,6 +4,12 @@ prodtype: ocp4
 
 title: 'Disable the AlwaysAdmit Admission Control Plugin'
 
+{{% set custom_jqfilter = '[.data."{{.var_openshift_kube_apiserver_config_data_name}}" | fromjson]' %}}
```

Comment:
It seems like from the log, `{{.var_openshift_kube_apiserver_config_data_name}}` did not get parsed

Reply:
We might want to wait for a CO upstream release